torch/nn/modules/linear.py

```diff
@@ -67,6 +67,8 @@ class Linear(Module):
         >>> print(output.size())
         torch.Size([128, 30])
     """
+    __constants__ = ['bias', 'in_features', 'out_features']
```

Comment:
Not related to this diff but: how did this work without constants previously? Should there have been a test failure somewhere that didn't happen?

Reply:
They're not actually used in `nn.Linear` except in `__repr__`